### PR TITLE
Add MailingComponent.php

### PR DESCRIPTION
**Pull request by m-hobby**
Overview
----------------------------------------
Include MailingComponent for searchkit.

Before
----------------------------------------
MailingComponent is not available

After
----------------------------------------
MailingComponent is available in dropdown

Technical Details
----------------------------------------

Comments
----------------------------------------
Required to adapt 'Headers, Footers, and Automated Messages' to searchkit

### CODE REVIEW

**Comment by colemanw** [comment]
@civicrm-builder add to whitelist
@19ATF72 thanks for this. Just one change: the file needs to be moved to `/ext/civi_mail`

**Comment by eileenmcnaughton** [comment]
style issue

image

**Comment by 19ATF72** [comment]
Fixed @colemanw @eileenmcnaughton

**Comment by eileenmcnaughton** [comment]
@19ATF72 the since would be 5.69 if merged today (5.70 from Thurs)

**Comment by 19ATF72** [comment]
Thanks @eileenmcnaughton, changed to 5.69.

File: ext/civi_mail/Civi/Api4/MailingComponent.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4;
+
+/**
+ * MailingComponent
+ *
+ * @searchable secondary
+ * @since 5.70
```
Comment (colemanw) [inline ext/civi_mail/Civi/Api4/MailingComponent.php:17]:
```suggestion
 * @since 5.69
```

**Comment by eileenmcnaughton** [comment]
test this please

**Pull request merged**